src/modules/tradeQuote/hooks/useTradeQuotePolling.ts

```diff
@@ -33,7 +33,7 @@ export function useTradeQuotePolling() {
     }
 
     const fetchQuote = () => {
-      updateQuoteState({ isLoading: true, error: null })
+      updateQuoteState({ isLoading: true })
```

Comment:
It was added here:
https://github.com/cowprotocol/cowswap/pull/2578/commits/d05ae0f5085685aaf07c78d2f42385522c0e051f

Actually, we shouldn't reset quote error **before** request, only after